client/helpers.ts

```diff
@@ -35,3 +35,11 @@ export const partitionLineItems = (items: Array<ILineConfig>) => partition(items
         return superdeskApi.helpers.assertNever(position);
     }
 });
+
+// Strip out any existing HTML tags and convert \n to <br> tags for HTML rendering
+export const cleanHtmlAndReplacePlaneNewLine = (text: string): string => {
```

Comment:
we have already stripHtmlTags in `core/utils`, and that split and joining wouldn't make reusable, that's specific to the usage, and imo can be avoided